config/config.go

```diff
@@ -1,11 +1,10 @@
 package config
 
 import (
-	"errors"
 	"fmt"
 	"github.com/micro/go-micro"
-	"gopkg.in/yaml.v2"
-	"io/ioutil"
+	"github.com/spf13/viper"
+	_ "github.com/spf13/viper/remote"
```

Comment:
a blank import should be only in a main or test package, or have a comment justifying it